### PR TITLE
#592 Add BarChat popover, change tooltips, add sun start and attemp c…

### PR DESCRIPTION
**Pull request by DzMakatun**
#592 Add run details to bar chart on clic
- Change chart tooltips to include more data
- Add run start time to monitoring
- Add run attempts counter (per infodate and version)
- Minor layout adjustments

### CODE REVIEW

File: menas/ui/components/dataset/datasetDetail.controller.js [hunk 1]
```diff
@@ -497,6 +495,40 @@ sap.ui.define([
...
+        this._oPopover.close();
+      }
+    },
+
+    onRecordsBarClick: function (oEvent) {
+      let oChart = oEvent.getSource().__chart;
+      let oActiveElement = oChart.getElementsAtEvent(oEvent.getParameters().event)[0];
+      if (!oActiveElement) return;
+      let runIndex = oActiveElement._index;
+      console.log(oEvent.getSource())
```
Comment (lokm01) [inline menas/ui/components/dataset/datasetDetail.controller.js]: Let's remove the logs pls

**Comment by Zejnilovic** [comment]
## 1 Can we make the popup centred for the view?
Right now the pop up from bar chart moves as I scroll up and down. Can it be shown always at the same place?
image
image
image

**Comment by Zejnilovic** [comment]
## 2 Close button for pop-up
I can already anticipate a question where is a close button. Can we add it?
Two possible placements
image

**Comment by DzMakatun** [comment]
Thank you, @Zejnilovic 
1.  Changed position to always at the top of the page
2. Added close button

**Pull request merged**